patches/minecraft/net/minecraft/world/item/AxeItem.java.patch

```diff
@@ -9,9 +9,9 @@
 -      Optional<BlockState> optional2 = Optional.ofNullable(HoneycombItem.f_150864_.get().get(blockstate.m_60734_())).map((p_150694_) -> {
 -         return p_150694_.m_152465_(blockstate);
 -      });
-+      Optional<BlockState> optional = Optional.ofNullable(blockstate.getToolModifiedState(level, blockpos, player, p_40529_.m_43722_(), net.minecraftforge.common.ToolActions.AXE_STRIP));
-+      Optional<BlockState> optional1 = Optional.ofNullable(blockstate.getToolModifiedState(level, blockpos, player, p_40529_.m_43722_(), net.minecraftforge.common.ToolActions.AXE_SCRAPE));
-+      Optional<BlockState> optional2 = Optional.ofNullable(blockstate.getToolModifiedState(level, blockpos, player, p_40529_.m_43722_(), net.minecraftforge.common.ToolActions.AXE_WAX_OFF));
++      Optional<BlockState> optional = Optional.ofNullable(blockstate.getToolModificationResult(p_40529_, net.minecraftforge.common.ToolActions.AXE_STRIP).toolModifiedState());
++      Optional<BlockState> optional1 = Optional.ofNullable(blockstate.getToolModificationResult(p_40529_, net.minecraftforge.common.ToolActions.AXE_SCRAPE).toolModifiedState());
++      Optional<BlockState> optional2 = Optional.ofNullable(blockstate.getToolModificationResult(p_40529_, net.minecraftforge.common.ToolActions.AXE_WAX_OFF).toolModifiedState());
```

Comment:
You forgot to regen patches, since this is does not compie.... `toolModifiedState` returns an optional, and `Optional.ofNullable(...toolModifiedState())` would as such be an `Optional<Optional<BlockState>>` not an `Optional<BlockState>`

Reply:
This didn't happen, *ever*.

Reply:
Gotya 😄